src/ui/configuration/nodeDebugConfigurationProvider.ts

```diff
@@ -14,7 +14,7 @@ import {
   AnyTerminalConfiguration,
   breakpointLanguages,
   ResolvingNodeConfiguration,
-  ResolvingTerminalConfiguration,
+  ResolvingTerminalConfiguration
```

Comment:
removing this caused the build to fail (prettier formatting is enforced in ci)

Reply:
Wonder why that happened 😕

Reply:
I have `formatOnSave` on with the `defaultFormatter` set to prettier. I think if you don't have the prettier extension installed, it'll use the TS language features which formats different